sync_diff_inspector/diff.go

```diff
@@ -435,3 +444,13 @@ func (df *Diff) Equal() (err error) {
 
 	return
 }
+
+// Judge if a table is in "exclude_tables" list
+func (df *Diff) InExcludeTables(exclude_tables []string, table string) bool {
+	for _, exclude_table := range exclude_tables {
+		if strings.EqualFold(exclude_table, table) {
```

Comment:
Should we need to support regex as we did for `Tables`?

Reply:
Well, for me, we just occasionally create a few tables in target-db that aren't  in source-db and they don't have common prefix/postfix. So maybe we can just make it simple.
Like to hear other's advices.